asv_bench/benchmarks/hdk/benchmarks.py

```diff
@@ -492,3 +492,6 @@ def time_groupby_agg_mean_dict(self, *args, **kwargs):
                 {col: "mean" for col in self.non_groupby_columns}
             )
         )
+
+
+from ..utils import setup  # noqa: E402, F401
```

Comment:
I'd add a comment why we need this

Reply:
Do you think that a comment inside `setup` function is not enough?

Reply:
yeah, it is fine, i think my main concern is that we would need to not forget to add this one in each new benchmarks.py (say, for unidist backend) which is a bit inconvenient - maybe make it default?

Reply:
I understand your concern, but however I don't know how to do it. Moreover, pandas also imports this function in every file. For now, this needs to be monitored.